### DISCUSSION TITLE
Parameters for Hub/Tip Loss Correction

### DISCUSSION THREAD

**Discussion by DmitryKudryavtsevA** [byuflowlab/FLOWUnsteady#94] [Q&A]
Tell me what rule to set the parameters t1, t2, t3 and h1, h2, h3?
Below in the description you make an adjustment using this code
```
import FLOWVLM as vlm
# No corrections
vlm.hubtiploss_nocorrection
# Original Prandtl corrections
vlm.hubtiploss_correction_prandtl
# Modified Prandtl with a strong hub correction
vlm.hubtiploss_correction_modprandtl
```

But in the rotorhover example [https://github.com/byuflowlab/FLOWUnsteady/blob/master/examples/rotorhover/rotorhover.jl](url)
you are using different data to adjust. 
Can you explain how to correctly set the parameters, and how to calculate them? Thank you!

**Comment by DmitryKudryavtsevA** [comment]
Can anyone tell me what rule is used to set the parameters t1,t2,t3 and h1,h2,h3?

**Reply by Turbulence1116** [reply]
Hey, I don't know exactly the rule for these parameters, but the author mentioned these here: 
https://github.com/byuflowlab/FLOWUnsteady/discussions/98#discussioncomment-7105272

**Reply by EdoAlvarezR** [reply]
I hope @Turbulence1116's reply helped! =]
